misc.py

```diff
@@ -0,0 +1,128 @@
+import os, sys
+import pygame
+
+
+colors = {'BLACK': (0, 0, 0),
+            'WHITE' : (255, 255, 255),
+            'GREEN' : (0, 255, 0),
+            'RED' : (255, 0, 0),
+            'BLUE' : (0, 0, 255),
+            }
+
+
+class Obstacle:
+    """A square obstacle, defined by its top left hand coordinate, length, and color.
+    Also takes in screen as an argument to draw the obstacle."""
+    def __init__(self, obs_x, obs_y, obs_len, screen, color):
+        """Initialize the instance."""
+        self.obs_x = obs_x              # top left hand x coordinate
+        self.obs_y = obs_y              # top left hand y coordinate
+        self.obs_len = obs_len          # side length
+        self.screen = screen            # game screen
+        self.color = color              # color of obstacle
+
+    def __repr__(self):
+        return 'Obstacle({}, {}, {}, {})'.format(self.obs_x, self.obs_y, self.obs_len, self.screen)
+
+    def moveForward(self):
+        """Update horizontal location and draw obstacle."""
```

Comment:
Python convention is `move_forward`, `is_gone`.

`move_forward_and_draw` is a clearer description of this methods behavior. Separate `move_forward` and `draw` might be more naturally units, though.